examples/language/lib/control_flow/branches.dart

```diff
@@ -0,0 +1,138 @@
+// ignore_for_file: unused_local_variable, prefer_typing_uninitialized_variables
```

Comment:
Let's remove this ignore so potential errors are still caught.

```suggestion
// ignore_for_file: unused_local_variable
```